multisig-wallet/project/contracts/multisig-contract/src/utils.sw

```diff
@@ -104,7 +106,7 @@ pub fn recover_signer(message_hash: b256, signature_info: SignatureInfo) -> b256
 ///
 /// * [b256] - The formatted message hash.
 fn eip_191_personal_sign_format(data_to_sign: b256) -> b256 {
-    let signed_data = encode_and_pack_signed_data(EIP191_INITIAL_BYTE, EIP191_VERSION_BYTE, data_to_sign);
+    let signed_data = encode_and_pack_signed_data(EIP191_INITIAL_BYTE.as_u64(), EIP191_VERSION_BYTE.as_u64(), data_to_sign);
```

Comment:
Since these are only used inside this file I would change the constants above to be `u64`.

Reply:
Its more readable to have a constant than a random byte